apps/web/public/static/locales/pt-BR/common.json

```diff
@@ -1939,5 +1939,26 @@
   "insights_user_filter": "Usuário: {{userName}}",
   "insights_subtitle": "Veja insights de reserva em seus eventos",
   "custom_plan": "Plano personalizado",
+
+  "event_limit_booking_limit_options_per_day": "Por Dia",
+  "event_limit_booking_limit_options_per_week": "Por Semana",
+  "event_limit_booking_limit_options_per_month": "Por Mês",
+  "event_limit_booking_limit_options_per_year": "Por Ano",
+  "phone": "Telefone",
+  "label_15_min": "15 minutes",
+  "label_30_min": "30 minutes",
+  "label_60_min": "60 minutes",
+  "nameless_team": "Equipe Sem Nome",
+  "event_host": "Responsável",
+  "event_attendee": "Participante",
+  "nameless": "Sem Nome",
+  "default_meeting": "Agenda Pessoa",
+  "default_slug": "agenda-pessoal",
+  "phone_call": "Attendee Phone Number",
```

Comment:
These next 3 should be translated, no?

Reply:
@thomasmazon thoughts here?

Reply:
About those 4, just miss on translation.